solutions/quickstart-vpc/variables.tf

```diff
@@ -0,0 +1,94 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud api token"
+  sensitive   = true
+}
+
+variable "prefix" {
+  type        = string
+  description = "Prefix for name of all resource created by this example"
+  validation {
+    error_message = "Prefix must begin and end with a letter and contain only letters, numbers, and - characters."
+    condition     = can(regex("^([A-z]|[a-z][-a-z0-9]*[a-z0-9])$", var.prefix))
+  }
+  default = "dev"
+}
+
+variable "region" {
+  type        = string
+  description = "Region where resources are created"
+}
+
+variable "use_existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The name of a new or the existing resource group to provision the cluster. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
+  default     = []
+}
+
+variable "ocp_version" {
+  type        = string
+  description = "Version of the OCP cluster to provision"
+  default     = null
+}
+
+variable "access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the resources created by the module."
+  default     = []
+}
+
+variable "ocp_entitlement" {
+  type        = string
+  description = "Value that is applied to the entitlements for OCP cluster provisioning"
+  default     = null
+}
+
+variable "number_worker_nodes" {
+  type        = number
+  description = "The number of workers to create in the cluster"
+  default     = 2
+}
+
+variable "machine_type" {
+  type        = string
+  description = "Worker node machine type. Use 'ibmcloud ks flavors --zone <zone>' to retrieve the list."
+  default     = "bx2.4x16"
+}
+
+variable "operating_system" {
```

Comment:
This is missing from the ibm_catalog.json? If you pull in the latest common-dev-assets and run pre-commit hooks, it will tell you all the missing vars from ibm_catalog.json.
You can add a dropdown box with the supported values then